custom_components/evnex/sensor.py

```diff
@@ -129,7 +129,7 @@ class EvnexChargerSessionCost(EvnexChargerEntity, SensorEntity):
         name="Charge Cost",
         icon="mdi:cash-multiple",
         # native_unit_of_measurement=CURRENCY_DOLLAR,
-        state_class=SensorStateClass.TOTAL_INCREASING,
+        state_class=SensorStateClass.TOTAL,
```

Comment:
My reading of https://developers.home-assistant.io/docs/core/entity/sensor/#available-state-classes is that perhaps measurement is more appropriate? What do you think?

I don't understand why `TOTAL_INCREASING` is impossible for money...

Reply:
We might need to update `last_reset` if we use `state_class=TOTAL`

Reply:
Looking at the error message I don't think `measurement` is an option either?! Only `None`. Not sure why, as you say `total_increasing` would make sense.